### ISSUE TITLE
Does VIRO support Google Street View?

### ISSUE THREAD

**Issue by EstaniPP** [viromedia/viro#857]
Hi, I want to create a VR app that includes Google Street View in order to simulate a walking tour around the city, I saw that Google offers a Javascript API and I want to know if it is possible to integrate it in my viro project.
In case this is possible does anyone have an example code?

Thank you very much.

**Comment by RoyceLeonD** [comment]
@EstaniPP Could you be a little bit more specific. What would you be using Viro for? 
From what I understand you want a React-based Google street view? 

If that is it, check out: https://reactjsexample.com/google-street-view-component-for-react-js/

Alternatively, If you are trying to integrate 3D objects (viro bject based mechanics), inside of a street view, then we need to think about object rendering and Google API extensions.

**Comment by EstaniPP** [comment]
What we are trying to do is integrate VIRO and Street View. We want to create an immersive experience using VR. We know the Maps App already has this feature but we couldn't find any API that allows us to do this.

**Comment by RoyceLeonD** [comment]
Hey @EstaniPP,

Sorry for the long delay. At the moment, I am not sure you can integrate "Viro" ,and "Google Street View". But there are solutions for what you are building. But first, let me rephrase what you want to build:

1. An interactive 2D map that the user can scroll, and move around.
2. When the user drops a pin on the map, it opens up a VR navigatable experience?

Also, I suggest if you might want to recruit some help from the Slack.

**Comment by EstaniPP** [comment]
Hi @RoyceLeonD , don't worry take your time, I really appreciate your help.

We want to develop an android app where a user selects a point A in the Google Map and then the app opens up a VR navigatable experience, once the user gets in it he will be able to walk around the city to reach a point B.

it seems more like the second option you gave us.

We want to use VIRO to create a VR experience and a google street view API to get the map and its information, so we can analize it and get the statistics. If you know an alternative way to create this without the use of VIRO it would also be helpful to us.

Thank you very much.

**Comment by RoyceLeonD** [comment]
Hi @EstaniPP ,

Again sorry for the delay. I have a couple of ideas, but it might be best discussed over slack instead of pinging the 1.1k stars for the codebase.

Just direct message me on the Viro Media Slack so I can respond better: https://viromedia.com/ (slack link at the bottom)

Until then I am going to close this post.
